lib/shrinkwrapper.js

```diff
@@ -1,10 +1,12 @@
+/* eslint no-sync: 0 */
```

Comment:
I think that is just easier to manage the flow for this because they are all file system operations for a single task.